tests/api/test_errors.py

```diff
@@ -44,23 +44,23 @@ def test_errors(problem_app):
 
     get_problem = app_client.get('/v1.0/problem')  # type: flask.Response
     assert get_problem.content_type == 'application/problem+json'
-    assert get_problem.status_code == 418
```

Comment:
`a2wsgi.ASGIMiddleware` uses the builtin `http` module underneath, which only supports status code 418 on Python 3.9+.